src/include/hip/rpp_hip_common.hpp

```diff
@@ -1459,158 +1459,265 @@ __device__ __forceinline__ void rpp_hip_rng_8_xorwow_f32(RpptXorwowState *xorwow
 
 // U8 loads for bilinear interpolation (4 U8 pixels)
 
-__device__ __forceinline__ void rpp_hip_interpolate1_bilinear_load_pln1(uchar *srcPtr, uint srcStrideH, float2 *locSrcFloor, float4 *srcNeighborhood_f4)
+
+__device__ __forceinline__ void rpp_hip_roi_range_check(float2 *locSrcFloor_f2, int4 *roiPtrSrc_i4, int2 *locSrc_i2)
```

Comment:
This function shouldn't be under the "// U8 loads for bilinear interpolation (4 U8 pixels)" category. Pls move this under the range-checks broad category at L138. Under that category, you can add a sub header for this function as "// ROI range check" or something.